plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx

```diff
@@ -75,14 +78,28 @@ export const PaymentGateways = () => {
 				};
 
 				const status = determineGatewayStatus();
+
 				return {
 					key: gateway.id,
 					title: (
 						<>
 							{ gateway.method_title }
-							<StatusBadge status={ status } />
+							{ hasIncentive ? (
+								<StatusBadge
+									status="has_incentive"
+									message={ __(
+										'Save 10% on processing fees',
+										'woocommerce'
+									) }
+								/>
+							) : (
+								<StatusBadge status={ status } />
+							) }
 						</>
 					),
+					className: hasIncentive
```

Comment:
Alternatively: (too nitpicky; use whatever suits your style)

```suggestion
					className: `woocommerce-list__item--payment-gateway ${ hasIncentive ?? 'has-incentive' }`
```

Reply:
I like the suggestion! I've applied it.